meson.build

```diff
@@ -733,12 +733,10 @@ if features['sdl2-gamepad']
     sources += files('input/sdl_gamepad.c')
 endif
 
-stdatomic_dep = cc.find_library('atomic', required: false)
+stdatomic_dep = cc.find_library('atomic', required: true)
 features += {'stdatomic': cc.has_header_symbol('stdatomic.h', 'atomic_int', dependencies: stdatomic_dep,
                                                required: get_option('stdatomic'))}
```

Comment:
Since `HAVE_STDATOMIC` is getting removed, there's no point in having this being added to the features. I guess the build should just error out completely if it fails at this step.